config/printer-creality-ender3-s1-2021.cfg

```diff
@@ -0,0 +1,132 @@
+# This file contains pin mappings for the stock 2021 Creality Ender 3
+# S1. To use this config, check the STM32 Chip on the V2.4S1 Board
+# then during "make menuconfig" select either the STM32F103 with a
+# "28KiB bootloader" or select the STM32F401 with a "64KiB bootloader"
+# and serial (on USB PA10/PA9) communication for both depending on the
+# STM32 Chip installed on your printers Motherboard.
+
+# If you prefer a direct serial connection, in "make menuconfig"
+# select "Enable extra low-level configuration options" and select
+# serial (on USB PA10/PA9), which is broken out on the 10 pin IDC
+# cable used for the LCD module as follows:
+# 3: Tx, 4: Rx, 9: GND, 10: VCC
```

Comment:
Just a query: Has this been verified in the S1 Pro with 401 cpus?

Reply:
why would it be different from the S1 pro with a 103 cpu? its the exact same display header why would the ports change at all that makes no sense.

Reply:
I'm testing it now and it doesnt seem to work.. I've connected the IDC pinouts and set the serial to usb pa10/pa9 but it didnt worked.

Reply:
I've just bypassed the CH340G chip and soldered the USART lines directly to my Pi. It seems to work now.